src/components/userProfile/RequestSent.js

```diff
@@ -67,7 +67,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     ...Platform.select({
       ios: {
-        marginTop: dimensions.heightWeight * 64,
+        marginTop: dimensions.heightWeight * 44 + 20,
```

Comment:
There is a convention that may helps your code better
You can refer [this](http://eslint.org/docs/rules/no-mixed-operators)
or just check out this code snippet
`
// This is difficult to read, not complicated.
const foo = 3 + 4 / 5 + 3 + 5 * 4 + 6

// This is much easier to read.
const foo = 3 + (4 / 5) + 3 + (5 * 4) + 6
`